drivers/synthetic_ce_assessment/app/models/synthetic_ce_assessment/enrollment_ce_assessment.rb

```diff
@@ -0,0 +1,71 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module SyntheticCeAssessment
+  class EnrollmentCeAssessment < ::GrdaWarehouse::Synthetic::Assessment
+    include ArelHelper
+
+    validates_presence_of :source
+
+    def assessment_date
+      source.entry_date
+    end
+
+    def assessment_location
+      source.project.project_id
+    end
+
+    def assessment_type
+      source.project.synthetic_ce_project_config&.assessment_type
+    end
+
+    def assessment_level
+      source.project.synthetic_ce_project_config&.assessment_level
+    end
+
+    def prioritization_status
+      source.project.synthetic_ce_project_config&.prioritization_status
+    end
+
+    def data_source
+      # TODO: is this correct? or should it be source.data_source or something else?
+      'Enrollment CE Assessment'
+    end
+
+    def self.sync
+      remove_orphans
+      add_new
+    end
```

Comment:
It's kind of confusing to me to have all these class methods on an AR model that aren't scopes or similar. I guess it's fine since this is a small class but if there's a chance it could grow, I'd suggest pulling this out into something like a job class

```
class SyntheticCeAssessmentSyncJob
  def perform
      remove_orphans
      add_new
  end
  # etc...
end
```

Reply:
I suppose these could be pulled out.  This is following the pattern of synthetic events and assessments where the logic for maintaining them is encapsulated within the active record model.  We don't expect this to grow in the future (though there may be more of them.)